<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;
background-color: rgb(255, 208, 142);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_menu">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔손글씨 펜</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_menu{
background-color: transparent;
border-image: url(:Menu_icon.png);
background: none;
background-repeat: none;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="menu">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>300</width>
      <height>900</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔손글씨 펜</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 227, 111)</string>
    </property>
    <widget class="QPushButton" name="pushButton_remove_menu">
     <property name="geometry">
      <rect>
       <x>202</x>
       <y>750</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_remove_menu{
background-color: transparent;
border-image: url(:before_icon.png);
background: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_calculator">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>배달의민족 주아</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 192, 119)</string>
     </property>
     <property name="text">
      <string>Calculator</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_curriculum_flowchart">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>배달의민족 주아</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 192, 119)</string>
     </property>
     <property name="text">
      <string>Curriculum Flowchart</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>배달의민족 주아</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_graduate_gak">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>배달의민족 주아</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 192, 119)</string>
     </property>
     <property name="text">
      <string>Graduate  gak</string>
     </property>
    </widget>
    <zorder>pushButton_remove_menu</zorder>
    <zorder>label</zorder>
    <zorder>pushButton_calculator</zorder>
    <zorder>pushButton_graduate_gak</zorder>
    <zorder>pushButton_curriculum_flowchart</zorder>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>590</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/승연/중대/2학년 2학기/객체지향프로그래밍(OOP)/팀플/character.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>들으면 졸업!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_bsm">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>BSM</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_engineering">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>공학주제</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_major_basic">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>전공기초</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_major_essential">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>전공필수</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_liberal_essential">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>필수교양</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_liberal_main">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>핵심교양</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_liberal_mach">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MACH교양</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_liberal_mach_practice">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MACH실습</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_bsm_score">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_engineering_score">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_major_basic_score">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_major_essential_score">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_liberal_essential_score">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_liberal_main_score">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_liberal_mach_score">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_liberal_mach_practice_score">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>학점,</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>학점,</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>학점,</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_23">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>학점,</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_24">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>학점,</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_25">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>학점,</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_26">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>학점,</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_27">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>학점</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/black_underline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/black_underline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/black_underline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/black_underline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/black_underline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/black_underline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/black_underline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>510</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/black_underline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_borderline">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-30</y>
      <width>897</width>
      <height>919</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/borderline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Username">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>님은</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/카카오톡 받은 파일/heart.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>520</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/party.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_borderline</zorder>
   <zorder>pushButton_menu</zorder>
   <zorder>label_pic</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_8</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_Username</zorder>
   <zorder>label_14</zorder>
   <zorder>label_13</zorder>
   <zorder>label_15</zorder>
   <zorder>menu</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="menu_logo.qrc"/>
 </resources>
 <connections/>
</ui>
